vosekast_control/TestSequence.py

```diff
@@ -55,7 +55,7 @@ def __init__(
                 self.logger.debug("Scale running, continuing.")
 
             # set fill to True
-            self.vosekast.constant_tank.fill_state = True
+            self.vosekast.constant_tank.state = "IS_FILLING"
```

Comment:
You can use here the propeties of the tank: 
```suggestion
            self.vosekast.constant_tank.state = self.vosekast.constant_tank.IS_FILLING
```